### PR TITLE
Add endpoint to expose service ids with assigned block ids

### PR DESCRIPTION
**Pull request by TsimurSh**
- added new endpoint /command/serviceIdsWithBlocks what retrieve service IDs with assigned block IDs

### CODE REVIEW

File: app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.Data;
+
+/**
+ * For outputting simple list of unsorted service IDs with lists of sorted block IDs
+ */
+@Data
+public class ApiServiceIdResponse {
+
+    @JsonProperty("serviceIds")
+    private List<ApiServiceId> apiServiceIds;
```
Comment (vesavlad) [inline app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java:18]: you could name directly the field as ```serviceIds```

File: app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Creates the API unsorted version of list of IDs.
+     *
+     * @param serviceIds
+     */
+    public ApiServiceIdResponse(Map<String, List<String>> serviceIds) {
+        apiServiceIds = new ArrayList<>();
+        serviceIds.forEach((key, list) -> apiServiceIds
```
Comment (vesavlad) [inline app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java:27]: just stream over the received serviceIds and map  them to the list needed to have a cleaner impl

File: app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java (hunk 1 above)
Comment (TsimurSh) [reply app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java]: ok. Corrected

File: app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java (hunk 2 above)
Comment (TsimurSh) [reply app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java]: Well done. @vesavlad  Please check it.

**Review by vesavlad — APPROVED** [review]

**Pull request merged**